src/Sylius/Bundle/ResourceBundle/Controller/RequestConfigurationFactory.php

```diff
@@ -74,12 +74,18 @@ private function parseApiParameters(Request $request)
     {
         $parameters = [];
 
-        if (preg_match(self::API_VERSION_REGEXP, $request->headers->get(self::API_VERSION_HEADER), $matches)) {
-            $parameters['serialization_version'] = $matches['version'];
+        $apiVersionHeaders = $request->headers->get(self::API_VERSION_HEADER, null, false);
+        foreach ($apiVersionHeaders as $apiVersionHeader) {
+            if (preg_match(self::API_VERSION_REGEXP, $apiVersionHeader, $matches)) {
+                $parameters['serialization_version'] = $matches['version'];
```

Comment:
You are right. I'm afraid jsm serializer won't resolve multiple versions properly. I would do the following. The last version should win, but rest should be concatenated.